internal/dataplane/sendconfig/sendconfig.go

```diff
@@ -66,7 +70,13 @@ func PerformUpdate(
 	}
 
 	promMetrics.RecordPushSuccess(metricsProtocol, duration, client.BaseRootURL())
-	log.Info("successfully synced configuration to kong")
+
+	if client.IsKonnect() {
+		log.Info("successfully synced configuration to Konnect")
+	} else {
+		log.Info("successfully synced configuration to Kong")
+	}
```

Comment:
An alternative that would fit both use cases and still holds true: `successfully synced configuration to Admin API` :)